Improve options for cross-compilation (#39)
zooba/pymsbuild: README.md

## Cross-compiling wheels

Cross compilation may be used by overriding the wheel tag, ABI tag,
or build platform, as well as the source for Python's includes and
libraries. These all use environment variables, to ensure that the
same setting can flow through a package installer's own process.

It is also possible to permanently override the wheel tag by adding

> a `'WheelTag'` metadata value, or the ABI tag by adding an `'ABITag'`

Author: ```suggestion
a `'WheelTag'` metadata value, or the ABI tag by adding an `'AbiTag'`
```

`ARM64`. Custom project files may use whatever they like.

```powershell

> # Directly specify the resulting wheel tags

Author: ```suggestion
# Directly specify the resulting wheel tag
```

# This is used for the wheel filename/metadata
$env:PYMSBUILD_WHEEL_TAG = "py38-cp38-win_arm64"

> # Directly set the ABI tag (overrides wheel tag)

Author: ```suggestion
# Directly set the ABI tag (or else taken from wheel tag)
```

# This is used for extension module filenames
$env:PYMSBUILD_ABI_TAG = "cp38-win_arm64"

> # Specify the Python platform (overrides ABI tag and wheel tag)

Author: ```suggestion
# Specify the Python platform (or else taken from ABI tag)
```
